### PR TITLE
Print the tests to run in the test-infra bazel job.

### PR DESCRIPTION
**Pull request by spxtr**
https://github.com/kubernetes/test-infra/pull/2414#issuecomment-293363552, I agree.

### CODE REVIEW

**Comment by spxtr** [comment]
This won't pass tests because we use the version from master, not the version from the PR...

**Comment by fejta** [comment]
Rewrite this as a scenario. Stop polluting test-infra with more bash scripts.

**Comment by spxtr** [comment]
Rather than fix forward I reverted.

File: jobs/pull-test-infra-bazel.sh [hunk 1]
```diff
@@ -40,18 +40,22 @@ buildables=$(bazel query \
...
+fi
 
 # Run affected tests.
 if [[ "${rc}" == 0 ]]; then
   tests=$(bazel query \
       --keep_going \
       --noshow_progress \
       "kind(test, rdeps(//..., set(${files[@]}))) except attr('tags', 'manual', //...)")
+  echo "bazel test ${tests} //verify:verify-all"
   bazel test --test_output=errors ${tests} //verify:verify-all && rc=$? || rc=$?
```
Comment (ixdy) [inline jobs/pull-test-infra-bazel.sh]: should we always be running verify-all?

Comment (spxtr) [inline jobs/pull-test-infra-bazel.sh]: We are.

Comment (spxtr) [inline jobs/pull-test-infra-bazel.sh]: We are.

Comment (ixdy) [inline jobs/pull-test-infra-bazel.sh]: I mean - is that the right thing? Or should we only run it when we need to?

Comment (spxtr) [inline jobs/pull-test-infra-bazel.sh]: I think we should always run them. If you change ANY source file then we need to run them, and I think it's safe to assume that a PR changes a source file.

@apelisse was looking into smarter ways to run these, but I'm not sure if anything came up.

**Comment by apelisse** [comment]
Forget about my change for now :-(.

Any file change would trigger the verify steps if we didn't exclude manual
targets, why do we do then?

Also, maybe it'd be interesting to print build targets, but ran test
targets are usually printed as a summary by bazel itself, aren't they?

**Pull request closed without merging**